addons/web/static/src/xml/base.xml

```diff
@@ -1327,8 +1327,8 @@
 
 <t t-name="GroupByMenuSelector">
     <li><select class="form-control oe-add-group oe-group-selector">
-        <t t-foreach="groupable_fields" t-as="field">
-            <option t-att-data-name="field"><t t-esc="groupable_fields[field].string"/></option>
+        <t t-foreach="groupable_fields_array" t-as="field">
```

Comment:
Creating another groupable_fields_array variable is a not really a good idea after all, because maybe someone will want to change some stuff on the fields so it will update groupable_fields, but he will never think to modify our new groupable_fields_array.

Wouldn't it work to t-foreach on "_.sortBy(_.values(this.groupable_fields), 'string')" ? This is the only place where we need them sorted after all.